Wabbajack/UI/MainWindow.xaml

```diff
@@ -8,8 +8,8 @@
     Title="Wabbajack"
     Width="1280"
     Height="960"
-    MinWidth="1280"
-    MinHeight="960"
+    MinWidth="1024"
```

Comment:
Still don' think this is a good idea but will check later

Reply:
Thank you erri, this way it won't affect the current behavior of the app, but it will work for people working on laptops that don't have full HD displays.